environment/cbs/uninstall_couchbase.py

```diff
@@ -0,0 +1,139 @@
+import json
```

Comment:
This is also redoing a bunch of orchestrator work, as well as doing some weird stuff with /etc/hosts....